include/powsybl/iidm/MultiVariantObject.hpp

```diff
@@ -30,6 +31,7 @@ class MultiVariantObject {
     virtual void reduceVariantArraySize(unsigned long number) = 0;
 
 private:
+    friend class Identifiable;
```

Comment:
Why do you need to make Identifiable friend of MultiVariantObject?

Reply:
So that MultiVariantObject protected methods can be called from their Identifiable implementation